components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/listener/InputValidationListener.java

```diff
@@ -71,10 +75,14 @@ public boolean doPreAddUser(String userName, Object credential, String[] roleLis
         if (!isEnable()) {
             return true;
         }
-        if (UserCoreUtil.getSkipPasswordPatternValidationThreadLocal()) {
-            return true;
+        Map<String, String> validationRequiredField = new HashMap<>();
```

Comment:
validationRequiredField > add a meaningful name and comments explaining the usage of the map

Reply:
Addressed